src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/Example1.java

```diff
@@ -18,7 +18,8 @@ public void foo() throws Exception {
       switch (i) {
         case 1:
           i++;
-        case 2: // violation 'Fall\ through from previous branch of the switch'
+            /* block */ /* fallthru */ // comment
+        case 2: // ok
```

Comment:
We discourage usage of ok comments, if you want to keep it, put some wording after "ok" to explain why no violation is expected here.

Apply to all other cases.

Reply:
Done @romani